src/UI/Implementation/ComponentRendererFSLoader.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/* Copyright (c) 2017 Richard Klees <richard.klees@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Implementation;
```

Comment:
Yes, I had this thought too, but wasn't sure either. The common prefix seems to be an indicator for namespace, but I wasn't sure whether these classes belong to Component or Renderer. I have no preference for either of the three choices, so your call.